sdk/impactreporting/Azure.ResourceManager.ImpactReporting/api/Azure.ResourceManager.ImpactReporting.net8.0.cs

```diff
@@ -0,0 +1,616 @@
+        Azure.ResourceManager.ImpactReporting.Models.Performance System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ImpactReporting.Models.Performance>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ImpactReporting.Models.Performance>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ImpactReporting.Models.Performance System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.Performance>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.Performance>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.Performance>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct Platform : System.IEquatable<Azure.ResourceManager.ImpactReporting.Models.Platform>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public Platform(string value) { throw null; }
+        public static Azure.ResourceManager.ImpactReporting.Models.Platform AzureMonitor { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.ImpactReporting.Models.Platform other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.ImpactReporting.Models.Platform left, Azure.ResourceManager.ImpactReporting.Models.Platform right) { throw null; }
+        public static implicit operator Azure.ResourceManager.ImpactReporting.Models.Platform (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.ImpactReporting.Models.Platform left, Azure.ResourceManager.ImpactReporting.Models.Platform right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct Protocol : System.IEquatable<Azure.ResourceManager.ImpactReporting.Models.Protocol>
```

Comment:
Same for these two types: https://github.com/Azure/azure-sdk-for-net/pull/48446/files#r1986671479